unattended.sh

```diff
@@ -98,6 +98,14 @@ if [ "x$?" '!=' x0 ] ; then
 fi
 print_step_end
 
+# Get the Centos version
+eval $(cat /etc/os-release | grep VERSION_ID)
```

Comment:
uname -mrs

Reply:
uname -r

Reply:
uname gives you information about the linux kernel version.

```
➜  ~ docker exec -ti 2d0 bash
[root@2d0b9de98725 /]# uname -r
4.19.121-linuxkit
[root@2d0b9de98725 /]# uname -mrs
Linux 4.19.121-linuxkit x86_64
```

We do want to know the version of Centos installed here

Reply:
Checking the file is easier as it is bash formatted variables so a simple eval gives you the value

```
[root@2d0b9de98725 /]# cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```